docs/assets/api-rapidoc.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html> <!-- Important: must specify -->
```

Comment:
Note to reviewer: download this file to try it locally it would work (appart the added menu urls)